.gitignore

```diff
@@ -37,5 +37,5 @@ log.txt
 cj
 
 # directory
-wiki/
+/wiki/

```

Comment:
This is a root path, not sure what it means in gitignore tbh, why was this change necessary?

Reply:
Because I added wiki folder in heartbeat which was getting ignored, so I had to change this to root path only as wiki git repository is cloned only here.

Reply:
ahhh I understand now!